WorkflowFunctionAdminBundle/Controller/Admin/WorkflowRightController.php

```
@@ -0,0 +1,66 @@
+<?php
+
+namespace OpenOrchestra\WorkflowFunctionAdminBundle\Controller\Admin;
+
+use OpenOrchestra\BackofficeBundle\Controller\AbstractAdminController;
+use OpenOrchestra\WorkflowFunction\Event\WorkflowRightEvent;
+use OpenOrchestra\WorkflowFunction\WorkflowRightEvents;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as Config;
+use Symfony\Component\HttpFoundation\Response;
+use OpenOrchestra\WorkflowFunction\Model\WorkflowRightInterface;
+
+/**
+ * Class WorkflowRightController
+ *
+ * @Config\Route("workflow-right")
+ */
+class WorkflowRightController extends AbstractAdminController
+{
+    /**
+     * @param Request $request
+     * @param string  $userId
+     *
+     * @Config\Route("/form/{userId}", name="open_orchestra_backoffice_workflow_right_form")
+     * @Config\Method({"GET", "POST"})
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_USER')")
+     *
+     * @return Response
+     */
+    public function formAction(Request $request, $userId)
+    {
+        $user = $this->get('open_orchestra_user.repository.user')->find($userId);
+        $workflowRight = null;
+        if (null !== $user) {
```

I think that if the user is Null you should throw an exception